### PR TITLE
Fix sidebar and add scrollbar

### PR DESCRIPTION
**Pull request by crandmck**
@christophherr This is largely based on your PR to this branch: #448.  I've tested it locally and it seems to work quite well, but thought I'd open a PR for your review and any other tweaks that might be needed.  PTAL!

### CODE REVIEW

**Comment by crandmck** [comment]
I'm going to go ahead and land this.  I don't expect any problems.... Thanks for your help @christophherr !

File: css/lavish-bootstrap.css [hunk 1]
```diff
@@ -717,7 +717,7 @@ pre code {
 .col-sm-12,
 .col-md-1,
 .col-md-2,
-.col-md-3,
+.col-md-3
```
Comment (damache) [inline css/lavish-bootstrap.css:720]: missing comma, think this is causing the sidebar to over extend into the page
screenshot from 2017-09-12 13-01-03

just happen within the hour.

Comment (crandmck) [reply css/lavish-bootstrap.css:720]: I checked in a correction for this.... See if that fixes it.

**Comment by damache** [comment]
no.  same thing.  I tried it in chrome too.  
screenshot from 2017-09-12 15-10-44

**Comment by damache** [comment]
I have a page open still before the change was pushed.  the computed size of the nav bar before is
screenshot from 2017-09-12 15-19-06

and after
screenshot from 2017-09-12 15-19-40

**Comment by crandmck** [comment]
Unfortunately, I can't reproduce the problem, even using FireFox. :-(

**Comment by christophherr** [comment]
@damache What is your screen size / screen resolution?
I get a similar result with zoom set to 80% (in Chrome)
http://prntscr.com/gkfpg4
We might just need to add more media queries, e.g.
```css

@media (min-width: 1410px) {
    .col-md-3 {
        width: 19%;
    }
}

@media (min-width: 1680px) {
    .col-md-3 {
        width: 17%;
    }
}

@media (min-width: 1880px) {
    .col-md-3 {
        width: 14%;
    }
}
```

**Comment by damache** [comment]
@christophherr @crandmck it's still happening and my browser is at 100%.  my screen is 1920 x 1080.  it happens when the sidebar is expanded and the slider appears.  
screenshot from 2017-09-13 09-49-23

notice in the image the sidebar is extended beyond the view

today I'm on my laptop and yesterday I was on my monitor at work.  think the same resolution

**Comment by crandmck** [comment]
OK, I can now reproduce this problem.  It occurs on FF only when you make your browser window REALLY wide, like 1800px+ ....  For me, it does not occur for widths less than that.  As a workaround, just make your browser window narrower.

**Pull request merged**